### ISSUE TITLE
Big Picture layout

### ISSUE THREAD

**Issue by ngraymon** [ngraymon/DNN-SE#9]
- what is beyond the scope
- all the extra moving pieces from the appendices
-

**Comment by ngraymon** [comment]
Primary target is to be able to produce a value for the wavefunction for a single system/model (to be decided on by #8)
An example of output is seen on pg 18 Fig 11, where the wavefunction is plotted first as a function of the distance (x) from the nuclei, and second as a function of the relative orientation (theta).

Example input structures are available on pg 19, Tables VII-XII


Goalposts towards calculating wavefunction:
- [] understanding the FNN (Fermi Neural Network) layout (pg3,13-15): see #7
- [] understanding the overall algorithm (pg3): see #7
- [] understanding the loss/optimization function (pg5-6): see #6
- []
- []


thoughts on simplification:
- first ipmlementation using only 1 or 2 determinants?
- make an artificaly H system with 1 proton and 1 electron, compute the energy using SOS and/or other methods (as in the paper)
and use this to verify our implementation works? This would allow us to evaluate the nuclei-electron part of the NN independent of the electron-electron part.
- the pretraining on the HF is something to be aware of
-
